src/LowercaseSentenceTokenizer.java

```diff
@@ -1,11 +1,14 @@
 import java.util.List;
 import java.util.Scanner;
+import java.util.ArrayList;
 
 /**
  * A tokenizer that converts text input to lowercase and splits it 
  * into a list of tokens, where each token is either a word or a period.
  */
 public class LowercaseSentenceTokenizer implements Tokenizer {
+  Scanner scanner = new Scanner(System.in);
+  List<String> tokenize = new ArrayList<>();
```

Comment:
These are not needed. The scanner gets passed as an argument to the instance, and you create your token List on line 36.